pkg/ovncontroller/volumes.go

```diff
@@ -2,6 +2,7 @@ package ovncontroller
 
 import (
 	"fmt"
+
```

Comment:
what is this extra line for?

Reply:
That has been the linter from the code editor, but I still think it is correct that separation for standard, current package, and vendor imports should be separated by a blank line.